cibyl/sources/elasticsearch/api.py

```diff
@@ -15,58 +15,59 @@
 """
 
 import logging
+from urllib.parse import urlsplit
 
 from cibyl.exceptions.elasticsearch import ElasticSearchError
+from cibyl.models.attribute import AttributeDictValue
+from cibyl.models.ci.build import Build
+from cibyl.models.ci.job import Job
+from cibyl.sources.elasticsearch.client import ElasticSearchClient
+from cibyl.sources.source import Source
 
 LOG = logging.getLogger(__name__)
 
 
-class ElasticSearchOSP:
+class ElasticSearchOSP(Source):  # pylint: disable=too-few-public-methods
```

Comment:
Jorge has set "min-public-methods=1" in .pylintrc so you probably don't need to disable it

Reply:
You're right. Thx.